client/my-sites/upgrades/checkout-thank-you/purchase-detail.jsx

```diff
@@ -8,22 +8,24 @@ import React from 'react';
  * Internal dependencies
  */
 import Button from 'components/button';
+import Gridicon from 'components/gridicon';
 
 const PurchaseDetail = ( {
-	additionalClass,
 	buttonText,
 	description,
 	href,
+	icon,
 	isPlaceholder,
 	target,
 	title
 } ) => {
-	const classes = classNames( 'checkout-thank-you__purchase-detail', additionalClass, {
+	const classes = classNames( 'checkout-thank-you__purchase-detail', {
 		'is-placeholder': isPlaceholder
 	} );
 
 	return (
 		<div className={ classes }>
+			{ icon && <div className="gridicon-wrapper"><Gridicon icon={ icon } size={ 72 } /></div> }
```

Comment:
This class should be namespaced to the component: `checkout-thank-you__gridicon-wrapper`.